src/test/v1.2/index.ts

```diff
@@ -34,7 +34,7 @@ function executeTests() {
 }
 
 function returnToTestSelection() {
-  location.reload();
+  location.href = '/v1.2/app/index.html';
```

Comment:
did you test this in Sail?

Reply:
In Sail location.reload() and location.href are not working.
Single test execution is still broken in Sail.
On clicking the button, it opens up the tests in Browser.

Reply:
anything to do with scoping? window.location and location are not necessarily the same thing depending on which scope you are in.